### ISSUE TITLE
v0.5.1 is incompatible with Consul Enterprise v1.13.3

### ISSUE THREAD

**Issue by v-rosa** [hashicorp/terraform-aws-consul-ecs#145]
I was testing the following demo: https://github.com/hashicorp/terraform-aws-hcp-consul/blob/main/examples/hcp-ecs-demo/main.tf

It relies on the [mesk-task module](https://github.com/hashicorp/terraform-aws-consul-ecs/tree/main/modules/mesh-task) of this repository to spin up an ECS service with all the needed side-cars.

Using Consul entreprise v1.13.3, its impossible to start the `consul-client` side car given the following error:

```log
==> failed to parse /consul/agent-defaults.hcl: 1 error occurred:
* invalid config key telemetry.disable_compat_1.9
```

Looking at the [release notes](https://developer.hashicorp.com/consul/docs/release-notes/consul/v1_13_x#what-s-changed), this paramter should be removed from v1.13 onwards. 

Given this [mesk-task module](https://github.com/hashicorp/terraform-aws-consul-ecs/tree/main/modules/mesh-task) is a dependency of the HCP feature which allows to bootstrap a Consul cluster with auto-generated terraform code, I would say it should fix by removing the parameter so users won't complain about a broken demo.

I can quickly open a PR to address it.

**Comment by v-rosa** [comment]
Never mind, its fixed already in main branch. Could you please release a hotfix for this?

**Comment by pglass** [comment]
Hi @v-rosa, 

Thanks for opening this issue! We've released v0.5.2 of Consul ECS which [adds support for Consul 1.13 and 1.14](https://developer.hashicorp.com/consul/docs/ecs/compatibility), so I'll close this out. Let us know if you run into any other  issues.
